apps/nextjs-website/react-components/types/MegaHeader/MegaHeader.types.ts

```diff
@@ -7,8 +7,11 @@ export interface MegaMenuItem {
     items: {
       label: string;
       href: string;
+      isNew: boolean;
+      badge?: string;
```

Comment:
```suggestion
      badge?: string;
```
Just check if badge exists, no need for another variable to do that.

Reply:
You're also already doing that, so isNew is never actually used.